static/app/actionCreators/modal.tsx

```diff
@@ -232,3 +234,5 @@ export async function openReprocessEventModal({
 
   openModal(deps => <Modal {...deps} {...options} />, {onClose});
 }
+
+export const getModalPortal = getModalPortalImport;
```

Comment:
I figured this was the best place to export `getModalPortal` since we are already exporting everything this from as `window.SentryApp.Modal`